src/Makefile

```diff
@@ -526,9 +556,24 @@ install: all
 	$(call MAKE_INSTALL,$(SERVER_NAME),$(INSTALL_BIN))
 	$(call MAKE_INSTALL,$(ENGINE_BENCHMARK_NAME),$(INSTALL_BIN))
 	$(call MAKE_INSTALL,$(ENGINE_CLI_NAME),$(INSTALL_BIN))
+
```

Comment:
I suggest skipping blank lines inside a make target body. Without blanks it's easier to see which lines belong to the same target.